CMakeLists.txt

```diff
@@ -43,22 +41,20 @@ install(
     DESTINATION include/rsl
 )
 install(
-    TARGETS rsl
-    EXPORT export_rsl
+    TARGETS rsl EXPORT rsl-targets
     ARCHIVE DESTINATION lib
     LIBRARY DESTINATION lib
     RUNTIME DESTINATION bin
-    INCLUDES DESTINATION include
+    INCLUDES DESTINATION include/rsl
 )
-
-ament_export_targets(export_rsl HAS_LIBRARY_TARGET)
-ament_export_dependencies(
-    Eigen3
-    fmt
-    rclcpp
-    tcb_span
-    tl_expected
+install(
+    EXPORT rsl-targets
+    NAMESPACE rsl::
+    DESTINATION ${CMAKE_INSTALL_DATADIR}
```

Comment:
@tylerjw We can replace this with whatever path the ament macros use. There are multiple places where these files can be installed. I think ament uses something like share/PROJECT_NAME/cmake or something like that.

Reply:
Fixed